packages/website/src/components/Modals/ModalForm.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { SubmitHandler, useForm } from 'react-hook-form';
+
+import { Button } from '@coderscamp/ui/components/Button';
+import { Flex } from '@coderscamp/ui/components/Flex';
+import { FormControl } from '@coderscamp/ui/components/FormControl';
+import { HelperText } from '@coderscamp/ui/components/HelperText';
+import { Input } from '@coderscamp/ui/components/Input';
+import { useBreakpointValue } from '@coderscamp/ui/hooks/useBreakpointValue';
+
+import { emailValidator } from '../Contact/Form';
+
+type ModalFormValues = {
+  name: string;
+  email: string;
+};
+
+const requiredValidator = {
+  required: 'To pole nie może być puste',
+};
+
+export const ModalForm = () => {
+  const size = useBreakpointValue({ base: 'md', sm: 'lg' } as const);
+  const {
+    register,
+    handleSubmit,
+    reset,
+    formState: { errors },
+  } = useForm<ModalFormValues>();
+
+  const onSubmit: SubmitHandler<ModalFormValues> = (data) => {
+    // eslint-disable-next-line no-console
+    console.log(data);
+    reset();
+  };
+
+  return (
+    <Flex
```

Comment:
💭 Maybe stack would be a little more suitable here as you want to achieve 16px of spacing between elements